01.w_Defects/wrong_arguments_func_pointer.c

```diff
@@ -550,7 +550,7 @@ void wrong_arguments_func_pointer_018_func_001(wrong_arguments_func_pointer_018_
 {
     memset(st, 0, sizeof(*st));
     st->a = 1;
-    func_pointer_018_global_set = 1;
+    func_pointer_018_global_set = 10;
```

Comment:
MAX is 10 at this point in the file, so it actually won't execute the line with the error unless this variable is also 10.